assets/js/components/settings/SettingsActiveModule/Header.js

```diff
@@ -75,8 +77,14 @@ export default function Header( { slug } ) {
 		if ( ! ( slug === 'analytics' && module?.connected ) ) {
 			return false;
 		}
-
-		return select( CORE_MODULES ).hasModuleOwnershipOrAccess( 'analytics' );
+		return (
+			select( CORE_MODULES ).hasModuleOwnershipOrAccess( 'analytics' ) ||
+			( ga4ReportingEnabled &&
+				isGA4Connected &&
+				select( CORE_MODULES ).hasModuleOwnershipOrAccess(
+					'analytics-4'
+				) )
+		);
```

Comment:
This change doesn't seem to be needed, as the resulting `hasAnalyticsAccess` value is only checked when `isGA4Connected` is falsy.

Reply:
Hi @techanvil. While this doesn't affect the UX, the selector still makes a call to the `check-access` endpoint and causes a network failure as well as a console error, as @wpdarren pointed out [here](https://github.com/google/site-kit-wp/issues/6861#issuecomment-1509291791).

We probably still need to address this in a new issue, post-bug bash.

CC: @aaemnnosttv

Reply:
Thanks for calling this out @nfmohit. Looks like it's subsequently been fixed:

https://github.com/google/site-kit-wp/blob/63e987b88946a8fff362f0b296488a35c5c23f2d/assets/js/components/settings/SettingsActiveModule/Header.js#L74-L80